packages/light-client-extension-helpers/src/smoldot/tasks.ts

```diff
@@ -0,0 +1,68 @@
+import { Client, AddChainError } from "."
```

Comment:
Re exporting the Schedule stuff in the API?

Personally I'd prob just ask for times as ms `number`s because that's pretty standard(eg for setTimeout, setInternal etc) and then keep Schedule 100% an implementation detail :)